### DISCUSSION TITLE
demo/markdow3n

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#761] [Announcements]
# demo/markdow3n

VuePress 主要从 Markdown 文件生成页面。因此，你可以使用它轻松生成文档或博客站点。 你应该创建和编写 Markdown 文件，以便 VuePress 可以根据文件结构将它们转换为不同的页面。

http://localhost:8080/demo/markdow3n.html

**Comment by bicycle808** [comment]
1111

**Reply by bicycle808** [reply]
222
